examples/stable_diffusion/train_dreambooth_loha.py

```diff
@@ -0,0 +1,1103 @@
+import argparse
```

Comment:
I only did a very superficial review of this script, as I assume that it's essentially the same as the corresponding LoRA script.

Reply:
That's true. I am almost certain that we could merge them into a single script.
However, each adapter has its own set of parameters, so to improve readability it should be better to create subparsers here for each adapter. Are you comfortable if we do it in a separate PR (maybe the one in which I will add LoKr)?

Reply:
I'm fine with having separate scripts. Merging them and adding sub-commands could be a nice-to-have feature, but not a must have.